packages/webapp/src/main.jsx

```diff
@@ -53,6 +53,7 @@ import consentSaga from './containers/Consent/saga';
 import callbackSaga from './containers/Callback/saga';
 import inviteUserSaga from './containers/InviteUser/saga';
 import exportSaga from './containers/ExportDownload/saga';
+import fieldWorkTaskSaga from './components/Task/FieldWorkTask/saga';
```

Comment:
I think this saga file should be somewhere in containers -- there are no other saga files in components that I can see.

Reply:
Changed the location and shifted to the containers folder